### DISCUSSION TITLE
Question about using .nosync files

### DISCUSSION THREAD

**Discussion by achpitt** [abraunegg/onedrive#1880] [Q&A] [answered]
Hello, I am trying to prevent certain folders from syncing locally by placing a .nosync file in the folder on OneDrive. The folder is still downloaded with both --check-for-nosync and check_nosync = "true". Do the .nosync files only work if they are placed in a local copy of the folder?

If that is the case, would the best course of action be to create the folder and its parents locally and place a .nosync file in it and then run the sync? I would prefer to use the .nosync files than set skip_dir in my config so I can move or rename the folder without concern that it will be synced.

Thanks!

**Comment by abraunegg** [comment, marked as answer]
@achpitt 
> I am trying to prevent certain folders from syncing locally by placing a .nosync file in the folder on OneDrive.

Unfortunately the `.nosync` files are only applicable for local folders - not remote due to how the API operates.

> If that is the case, would the best course of action be to create the folder and its parents locally and place a .nosync file in it and then run the sync?

You would only really need to create the parent locally, as any child would be skipped.

The alternate here is to use 'sync_list' to control what to include|exclude from syncing - outside of configuring 'skip_dir'
